docs/quickstart.rst

```diff
@@ -13,7 +13,7 @@ create a minimal application, or use its scaffolding tool:
 
 ::
 
-    pcreate -s starter MyProject
+    $ pcreate -s starter MyProject
```

Comment:
I thought so, but I realize they give the context implicity. One knows that this is a command to run in a shell.

We can remove them if we don't agree :)